### ISSUE TITLE
@JsonIgnoreProperties annotation not respected

### ISSUE THREAD

**Issue by helloariel** [springfox/springfox#113]
I use @JsonIgnoreProperties to omit some fields when my models are serialized, but it seems swagger doesn't take them into account and adds these fields in the generated specs anyway.

**Comment by codependent** [comment]
@dilipkrish I am having the same problem as @agdula: properties from a parent class show up in the Model/Model schema despite being in the `@JsonIgnoreProperties` annotation.

What can we do in these case?

**Comment by dorgla** [comment]
problem still exists in 2.4.0
@JsonIgnoreProperties({ "field1" , "field2" }) is ignored for the model when used inside an Object (on a field which is an object itself) and using this object as response model.
not be considered in the model. However the serialization works as expected.

Example Pseudo-Code:
```java
public class MyClass {
	private String someString;
	@JsonIgnoreProperties({"address","fullname"})
	private UserEntity user;
...
```
UserEntity
```java
public class UserEntity{
	private String address;
	private String fullname;
	private String nickname;
...
```

Using for REST Documentation like
```java
@ApiResponses(value = {
			@ApiResponse(code = 200, message = "Successful retrieval of user", response = MyClass .class),
...
```

**Comment by lhanson** [comment]
+1, please reopen the issue

**Comment by duckladydinh** [comment]
gdfd

**Comment by duckladydinh** [comment]
Please reopen the issue. The problem is if @JsonIgnoreProperties is not respected, then when we generate the swagger documentation, the JSON data and the swagger model will not match. That's a serious issue, right?

**Comment by dilipkrish** [comment]
@duckladydinh this is an old issue. Has [this expectation](https://github.com/springfox/springfox/issues/113#issuecomment-22653341) changed?

**Comment by dilipkrish** [comment]
@Raniz85 thanks for the explanation. Would you mind creating a new issue for it, so it can be tracked

**Comment by distelka** [comment]
Any action on this ?

**Comment by rameshkrishnanme** [comment]
Is it not fixed still ? facing the same issue. Please suggest workaround

**Comment by arielalvesdutra** [comment]
I have the same problem: @JsonIgnoreProperties("someAttribute") is not ignoring the property to the documentation. @JsonIgnore works, but then is all the attribute ignored.

**Comment by ChinmayanP** [comment]
Is it fixed?  I am facing the same problem in the 2.9.2 version.

**Comment by ecalado** [comment]
The problem persist in the 2.9.2 version. Is there another issue to fix it? Or something that I can do using  this annotation?

**Comment by Jojoooo1** [comment]
Any new  or workaround found ?

**Comment by dilipkrish** [comment]
This should be fixed in 3.0.0

**Comment by samshay** [comment]
> This should be fixed in 3.0.0

I test in version 3.0.0 but i'v got the same problem.
